editor/editor_sectioned_inspector.cpp

```diff
@@ -308,7 +308,7 @@ EditorInspector *SectionedInspector::get_inspector() {
 }
 
 SectionedInspector::SectionedInspector() :
-		obj(-1),
+		obj(0),
```

Comment:
As I mentioned in the previous comment, there's another line with `obj = -1` in this file.